docs/data/charts/components/SeriesDemo.js

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { ChartsXAxis } from '@mui/x-charts/ChartsXAxis';
+import { ChartsYAxis } from '@mui/x-charts/ChartsYAxis';
+import { LinePlot } from '@mui/x-charts/LineChart';
+import { useLineSeries, useXAxis, useXScale, useYScale } from '@mui/x-charts/hooks';
+import { ChartsSurface } from '@mui/x-charts/ChartsSurface';
+import { ChartDataProvider } from '@mui/x-charts/ChartDataProvider';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+function ExtremaLabels({ seriesIndex }) {
+  const lineSeries = useLineSeries();
+  const xAxis = useXAxis('x');
+
+  if (!lineSeries) {
+    return null;
+  }
+
+  const seriesId = lineSeries.seriesOrder[seriesIndex];
+  const series = lineSeries.series[seriesId];
+
+  const min = series.data.reduce(
+    (a, b) => Math.min(a ?? Infinity, b ?? Infinity),
+    Infinity,
+  );
+  const max = series.data.reduce(
+    (a, b) => Math.max(a ?? -Infinity, b ?? -Infinity),
+    -Infinity,
+  );
+
+  return (
+    <React.Fragment>
+      {series.data.map((y, index) => {
+        const x = xAxis.data?.[index];
+
+        if (x == null || y == null) {
+          return null;
+        }
+
+        if (y !== min && y !== max) {
+          return null;
+        }
+
+        return (
+          <PointLabel
+            x={x}
+            y={y}
+            placement={y === min ? 'below' : 'above'}
+            color={series.color}
+          />
+        );
+      })}
+    </React.Fragment>
+  );
```

Comment:
We're iterating over the data 3 times, but I think this is more readable than a more efficient algorithm. Open to change, if you think a more performant solution would be better here.

Reply:
shouldn't be an issue in this example.

Reply:
Instead of `a` and `b` could we use more meaning full variable like `min` (or `acc`) and `value`. Otherwise it looks like you're handling two values at the same time.

But I agree with JC it's not an issue :+1:

Reply:
I couldn't use `min` or `max` because it would show a lint error since there's already `min` and `max` in the outer scope, so I changed those to `acc` and `value` 👍